mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/model/Attachment.java

```diff
@@ -35,11 +36,12 @@ public static Builder builder() {
         return new Builder();
     }
 
-    public static Attachment from(byte[] bytes, String type) {
+    public static Attachment from(byte[] bytes, String type, Optional<String> name) {
```

Comment:
I would say this method have now too much argument, and calls becomes hard to read. I'd rather see calls to the builder...